test/model/src/core/verify.cpp

```diff
@@ -100,6 +100,12 @@ kv_table_verifier::verify(WT_CONNECTION *connection)
 
     /* Get the model cursor. */
     kv_table_verify_cursor model_cursor = _table.verify_cursor();
+    model_cursor.set_checkpoint(ckpt);
+
+    /* Create the WiredTiger cursor config. */
+    std::string cursor_config;
+    if (ckpt)
+        cursor_config += std::string("checkpoint=") + ckpt->name();
```

Comment:
Can we move this closer to where it is used? Line 120.

Reply:
Sure.